bin/run-tests

```diff
@@ -37,24 +51,39 @@ main() {
     # Migrate and seed db
     echo
     log "Migrating & seeding db"
-    docker exec test-backend bash -c "yarn db:migrate"
-    check_exit "$?"
-    docker exec test-backend bash -c "yarn db:seed;"
-    check_exit "$?"
+    if [[ $docker == "true" ]]; then
+      docker exec test-backend bash -c "yarn db:migrate"
+      check_exit "$?"
+      docker exec test-backend bash -c "yarn db:seed"
+      check_exit "$?"
+    else
+      yarn db:migrate
+      check_exit "$?"
+      yarn db:seed
+      check_exit "$?"
+    fi
 
     if [[ "$opt" == "backend" || -z "$opt" ]]; then
         # Test backend
         echo
         log "Running backend tests"
-        docker exec test-backend bash -c "yarn test:ci"
+        if [[ $docker == "true" ]]; then
+          docker exec test-backend bash -c "yarn test:ci"
+        else
+          yarn test:ci
```

Comment:
If we aren't using `test-backend` or `test-frontend`, would we also want to alter the `docker-compose up` command to specify that we only want to bring up the `test-db` service?